mindsdb/api/http/namespaces/sql.py

```diff
@@ -24,6 +25,14 @@
 @ns_conf.route("/query")
 @ns_conf.param("query", "Execute query")
 class Query(Resource):
+    meter = otel_metrics.metrics.get_meter("query_service_meter")
```

Comment:
We need to double check this, I think it will work only when using SQL Editor and Python SDK. If the user is using the `MySQL API` this will not work.

Reply:
I changed where I added this metric to inside the SQL executor.